R/control.R

```diff
@@ -92,10 +95,19 @@ spim_control <- function(short_run, n_chains, deterministic = FALSE,
                                   path = mcmc_path)
 
   n_threads <- parallel$n_threads_total / parallel$n_workers
+
+  # Force severity to be FALSE if running multiregion fits, as severity
+  # weight calculation does not currently work with multiregion = TRUE
+  if (multiregion) {
+    severity <- FALSE
+  }
+
   particle_filter <- list(n_particles = n_particles,
                           n_threads = n_threads,
                           seed = NULL,
-                          compiled_compare = compiled_compare)
+                          compiled_compare = compiled_compare,
+                          severity = severity,
+                          multiregion = multiregion)
```

Comment:
`multiregion` not needed here with change suggested https://github.com/mrc-ide/spimalot/pull/208/files#r857780215